From d3390124d8c69543185506246ef6e0a0eb5dbb18 Mon Sep 17 00:00:00 2001
From: Lu Yahan <yahan@iscas.ac.cn>
Date: Mon, 19 Jul 2021 09:55:43 +0800
Subject: [PATCH] [riscv64] Skip test case due to timeout in buildbot

- msjunit/compiler/regress-1226988

Bug: v8:12002
Change-Id: I413bb54d64caf33db5493c69dff83a7ba09810ea
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/3035263
Reviewed-by: Jakob Gruber <jgruber@chromium.org>
Commit-Queue: Jakob Gruber <jgruber@chromium.org>
Auto-Submit: Yahan Lu <yahan@iscas.ac.cn>
Cr-Commit-Position: refs/heads/master@{#75773}
---
 test/mjsunit/mjsunit.status | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/mjsunit/mjsunit.status b/test/mjsunit/mjsunit.status
index dab812cbe59..bd550eb07e9 100644
--- a/test/mjsunit/mjsunit.status
+++ b/test/mjsunit/mjsunit.status
@@ -822,6 +822,7 @@
   'regress/regress-490': [SKIP],
   'regress/regress-create-exception': [SKIP],
   'regress/regress-3247124': [SKIP],
+  'compiler/regress-1226988': [SKIP],
 
   # Requires bigger stack size in the Genesis and if stack size is increased,
   # the test requires too much time to run.  However, the problem test covers
-- 
2.35.1

